sdk/component/store/remote/postgresql/sqlc/migrations/20241209103854_findings_table.sql

```diff
@@ -0,0 +1,8 @@
+-- create "findings" table
+CREATE TABLE IF NOT EXISTS finding (
+   id SERIAL PRIMARY KEY,
+   instance_id UUID NOT NULL UNIQUE,
+   findings JSONB NOT NULL,
```

Comment:
This is loaded as an array,  On a findings level this should just be one json object.  The column name should be renamed to something like "finding_details"

Reply:
Good spot - I'll fix this

Reply:
Fixed.